### PR TITLE
[PT FE] Fix aten::conv2d conversion issues related to bias

### PR DESCRIPTION
**Pull request by mmikolajcz**
### Details:
 - *Fix issue with conversion due to missing bias shape in constant*
 - *...*

### Tickets:
 - *101077*

### CODE REVIEW

File: src/frontends/pytorch/src/op/convnd.cpp [hunk 1]
```diff
@@ -50,7 +50,7 @@ OutputVector translate_convnd(const NodeContext& context) {
                                                       pad_type);
     }
     if (!context.input_is_none(2)) {
-        auto bias = context.get_input(2);
+        auto bias = context.get_input_from_visible_context(2);
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/convnd.cpp:53]: We can do this only when bias is constant. Please add such verification.

Comment (mmikolajcz) [reply src/frontends/pytorch/src/op/convnd.cpp]: Added cast to Constant as condition to check whether bias can be acquired from visible context

File: src/frontends/pytorch/src/op/convnd.cpp [hunk 2]
```diff
@@ -51,6 +51,9 @@ OutputVector translate_convnd(const NodeContext& context) {
     }
     if (!context.input_is_none(2)) {
         auto bias = context.get_input(2);
+        if (std::dynamic_pointer_cast<v0::Constant>(context.get_input_from_visible_context(2).get_node_shared_ptr())) {
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/convnd.cpp:54]: Would be better to call `get_input_from_visible_context` once also please add at least one test

Comment (mmikolajcz) [reply src/frontends/pytorch/src/op/convnd.cpp]: Done

**Pull request merged**